include/mbedtls/cipher.h

```diff
@@ -69,102 +69,102 @@ extern "C" {
 #endif
 
 /**
- * \brief     An enumeration of supported ciphers.
+ * \brief     Supported cipher types.
  *
  * \warning   ARC4 and DES are considered weak ciphers and their use
- *            constitutes a security risk. We recommend considering stronger
- *            ciphers instead.
+ *            constitutes a security risk. We recommend you consider using
+ *            stronger ciphers instead.
  */
 typedef enum {
-    MBEDTLS_CIPHER_ID_NONE = 0,
-    MBEDTLS_CIPHER_ID_NULL,
-    MBEDTLS_CIPHER_ID_AES,
-    MBEDTLS_CIPHER_ID_DES,
-    MBEDTLS_CIPHER_ID_3DES,
-    MBEDTLS_CIPHER_ID_CAMELLIA,
-    MBEDTLS_CIPHER_ID_BLOWFISH,
-    MBEDTLS_CIPHER_ID_ARC4,
+    MBEDTLS_CIPHER_ID_NONE = 0,  /**< None. */
```

Comment:
These comments are superfluous.

Reply:
Any item should be documented, even if you think it's obvious. If the description is inaccurate, please advise.

Reply:
We had some discussion on the previous wording, including @sbutcher-arm, so I'd prefer him giving his consent to the change here, too.

Reply:
Waiting for an answer on this...

Reply:
Suggestion: "Placeholder to mark the end of cipher ID lists."

Reply:
Acceptable to me. changed.